python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -974,5 +944,41 @@ def applyModelWeights(self, modelImages, refImage, modelWeights):
         """
         if self.config.useModelWeights:
             for model in modelImages:
-                model.image.array *= modelWeights
-                model.image.array += refImage.image.array*(1. - modelWeights)/self.config.dcrNumSubfilters
+                model.array *= modelWeights
+                model.array += refImage.array*(1. - modelWeights)/self.config.dcrNumSubfilters
+
+    def loadSubExposures(self, bbox, statsCtrl, tempExpRefList, imageScalerList, spanSetMaskList):
+        """Pre-load sub-regions of a list of exposures.
+
+        Parameters
+        ----------
+        bbox : `lsst.afw.geom.box.Box2I`
+            Sub-region to coadd
+        statsCtrl : `lsst.afw.math.StatisticsControl`
+            Statistics control object for coadd
+        tempExpRefList : `list` of `lsst.daf.persistence.ButlerDataRef`
+            The data references to the input warped exposures.
+        imageScalerList : `list` of `lsst.pipe.task.ImageScaler`
+            The image scalars correct for the zero point of the exposures.
+        spanSetMaskList : `list` of `dict` containing spanSet lists, or None
+            Each element is dict with keys = mask plane name to add the spans to
```

Comment:
Please expand the docstring for `spanSetMaskList` a bit: I'm having trouble parsing this.